docs/sources/enterprise/configuration.md

```diff
@@ -0,0 +1,187 @@
++++
+title = "Enterprise configuration"
+description = "Enterprise configuration documentation"
+keywords = ["grafana", "configuration", "documentation", "enterprise"]
+type = "docs"
+[menu.docs]
+name = "Enterprise configuration"
+identifier = "enterprise-config"
+parent = "enterprise"
+weight = 300
++++
+
+# Grafana Enterprise configuration
+
+This page describes Grafana Enterprise-specific configuration options that you can specify in a `.ini` configuration file or using environment variables. Refer to [Configuration]({{< relref "../installation/configuration.md" >}}) for more information about available configuration options.
+
+## [white_labeling]
+
+### app_title
+
+Set to your company name to override application title.
+
+### login_logo
+
+Set to complete url to override login logo.
```

Comment:
```suggestion
Set to complete URL to override login logo.
```

Reply:
URL should always be capitalized except in code where it would break things. Apply throughout.